src/ripple/app/ledger/impl/LedgerMaster.cpp

```diff
@@ -211,6 +211,7 @@ LedgerMaster::setValidLedger(
             !app_.getMaxDisallowedLedger() ||
             l->info().seq + max_ledger_difference_ >
                     app_.getMaxDisallowedLedger());
+    (void) max_ledger_difference_;
```

Comment:
Not necessary but it might be helpful to add a comment describing why this is here.